test/Libraries/Microsoft.Extensions.AI.AotCompatibility.TestApp/Program.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma warning disable CA1031 // Do not catch general exception types
+#pragma warning disable S108 // Nested blocks of code should not be left empty
+#pragma warning disable S125 // Remove this commented out code
```

Comment:
?

1. Are these all necessary?
2. If we don't care, should we be turning them off in an .editorconfig?

Reply:
> Are these all necessary?

The last one is. The first two are leftover from a previous incarnation of the code in this file and can be removed.

> If we don't care, should we be turning them off in an .editorconfig?

There are a bunch of dubious rules enabled for the whole repo, but I've not wanted to inflict my opinions on what was already established for the repo, and so we've just been disabling rules in M.E.AI per project or file as needed.